src/components/grid-aware/Footer/Footer.module.css

```diff
@@ -0,0 +1,257 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-rows: 120px 380px 78px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    text-align: center;
+  }
+
+  .gridParent {
+    column-gap: var(--grid-column-gap);
+    display: grid;
+    grid-template-rows: 190px auto auto;
+  }
+}
+
+/* Top Row */
+
+.gridTopRow {
+  align-content: center;
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  grid-row: 1;
+  justify-content: center;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  align-self: center;
+  display: flex;
+  justify-content: center;
+}
+
+.subscribeMessage {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  padding-right: 20px;
+  width: 340px;
+}
+
+.gridTopRow a {
+  padding: 14px 40px;
+}
```

Comment:
This is probably not a good idea, as it is relying on internal details about the Button component (the fact that it's an `<a>` tag), and it could possibly end up selecting something else that happened to be an `<a>` tag in this component if we added more stuff in the future.

Also, we'll probably actually want to make this an `<input type="submit">` element and wrap both this and the text input in a `<form>`, since that's semantically what these things are. It might be worth defining a separate React component for the `<input type="submit">` version of this component, where the height is matched to the `<input type="text">` element.

Reply:
Unable to modify Button component from a parent element because the classname is hashed